src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -24,12 +25,47 @@ const GroupTabDetailsWrapper = ({
   hasEdgeImages,
 }) => {
   const [tab, setTab] = useState(0);
+
   const { hasAccess: canViewHosts } = usePermissionsWithContext(
     REQUIRED_PERMISSIONS_TO_READ_GROUP_HOSTS(groupId)
   );
+  const conventionalSystemsContent = (
+    <GroupSystems
+      groupName={groupName}
+      groupId={groupId}
+      hostType={hybridInventoryTabKeys.conventional.key}
+    />
+  );
+
+  const immutableSystemsContent = (
+    <GroupImmutableSystems
+      groupId={groupId}
+      groupName={groupName}
+      hostType={hybridInventoryTabKeys.immutable.key}
+    />
+  );
+
+  const [component, setComponent] =
+    activeTab == hybridInventoryTabKeys.conventional.key
+      ? useState(conventionalSystemsContent)
+      : useState(immutableSystemsContent);
```

Comment:
Conditional hooks call is discouraged and leads to worse performance in React: https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks. I would like to have this updated in a way that we always call hooks in the same order.

Reply:
thanks!I've change it